packages/builders/scripts/vats/terminate-governor-instance.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * @file Terminate price-feed governor instances such as mainnet v110.
+ */
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+import { E } from '@endo/far';
+
+const SELF = '@agoric/builders/scripts/vats/terminate-governor-instance.js';
+const USAGE =
+  'Usage: agoric run /path/to/terminate-governor-instance.js <$boardID:$instanceKitLabel>...';
+
+const repr = val =>
+  typeof val === 'string' || (typeof val === 'object' && val !== null)
+    ? JSON.stringify(val)
+    : String(val);
+const defaultMakeError = (strings, ...subs) =>
+  Error(
+    strings.map((s, i) => `${i === 0 ? '' : repr(subs[i - 1])}${s}`).join(''),
+  );
+const makeUsageError = (strings, ...subs) => {
+  const err = defaultMakeError(strings, ...subs);
+  console.error(err.message);
+  console.error(USAGE);
+  return err;
+};
+
+const rtarget = /^(?<boardID>board[0-9]+):(?<instanceKitLabel>.+)$/;
+/**
+ * @param {string[]} args
+ * @param {(strings: TemplateStringsArray | string[], ...subs: unknown[]) => Error} [makeError]
+ * @returns {Array<{boardID: string, instanceKitLabel: string}>}
+ */
+const parseTargets = (args, makeError = defaultMakeError) => {
+  /** @type {Array<{boardID: string, instanceKitLabel: string}>} */
+  const targets = [];
+  const badTargets = [];
+  for (const arg of args) {
+    const m = typeof arg === 'string' && arg.match(rtarget);
+    if (!m) {
+      badTargets.push(arg);
+    } else {
+      // @ts-expect-error cast
+      targets.push(m.groups);
+    }
+  }
+  if (badTargets.length !== 0) {
+    throw makeError`malformed target(s): ${badTargets}`;
+  }
+  return targets;
```

Comment:
I think I'd like this to fail if `targets.length === 0`.

Reply:
Nice catch.